README.md

```diff
@@ -172,7 +171,7 @@ Reminder: the 'countryAndRegions' prop that can be sent in the ReactFormBuilder
 ```
 
 ### Custom list country example:
-```yaml
+```json
```

Comment:
nit: I think this might be `js` or `jsx`, based on the lack of `""`.

Reply:
Solved [here](https://github.com/onebeyond/react-form-builder/commit/bddcea462988cfa28833df9e2df7f7599be6e8d0)